### ISSUE TITLE
[BUG] Turning on the rolling machine causes game crash

### ISSUE THREAD

**Issue by Mister-Bones1501** [zmaster587/AdvancedRocketry#2250]
[Please fill out the form below and delete the sections in square brackets after reading them]

## Version of Advanced Rocketry
>1.7.0-232

## Have you verified this is an issue in the latest unstable build
-  N

## Version of LibVulpes
>0.4.2.-84

## Version of Minecraft
>1.12.2

## Does this occur without other mods installed
-  Yes (my main world with mods crashed so I made a test world with only advanced rocketry and libvulpes and it crashed too).

> Crash report -  https://pastebin.com/uECjfeNq

## Description of the problem
> I can form the rolling machine and it generates the multi block. it also receives power and fluid. I place an ingot into the input hatch and then go to the rolling machine interface. When I click the power switch in the GUI the game freezes, the screen turns into the Minecraft dirt loading screen,  words appear saying "shutting down internal server," then the launcher crashes, and finally a report pops up.

**Comment by Mister-Bones1501** [comment]
I tired an iron ingot and then a gold one. both caused crashed.

**Comment by Mister-Bones1501** [comment]
</Recipe
<Recipe timeRequired="300" power ="200"
<input
<oreDict>plateIron</oreDict
<fluidStack>water;100</fluidStack
</input
<output
<itemStack>libvulpes:productsheet;1;1</itemStack
</output
</Recipe

here's what the recipe says for a plate

**Comment by Mister-Bones1501** [comment]
hello, so it tried a few things in the mod.

All machines work except the rolling machine

I set up the machine again with power and turned on and off no problem (I was not processing anything). then i filled it up with water buckets. I tried turning it off and on (no processing) and it crashed. so maybe fluid input is the issue?

heres the crash report but only when fluid was added. ( i am not entirely sure how to read code, but i am learning). 
https://pastebin.com/bcyauLe3
